--- google-chrome-15.0.874.106/google-chrome.desktop~	2011-10-27 00:22:10.000000000 +0300
+++ google-chrome-15.0.874.106/google-chrome.desktop	2011-10-27 00:24:02.861899987 +0300
@@ -105,12 +105,12 @@
 Comment[zh_CN]=访问互联网
 Comment[zh_HK]=連線到網際網路
 Comment[zh_TW]=連線到網際網路
-Exec=/usr/bin/google-chrome-stable %U
+Exec=/usr/bin/google-chrome %U
 Terminal=false
 Icon=google-chrome
 Type=Application
 Categories=Network;WebBrowser;
-MimeType=text/html;text/xml;application/xhtml_xml;image/webp;x-scheme-handler/http;x-scheme-handler/https;x-scheme-handler/ftp;
+MimeType=text/html;text/xml;application/xhtml+xml;image/webp;x-scheme-handler/http;x-scheme-handler/https;x-scheme-handler/ftp;
 X-Ayatana-Desktop-Shortcuts=NewWindow;NewIncognito
 
 [NewWindow Shortcut Group]
@@ -165,7 +165,7 @@
 Name[vi]=Cửa sổ Mới
 Name[zh_CN]=新建窗口
 Name[zh_TW]=開新視窗
-Exec=/usr/bin/google-chrome-stable
+Exec=/usr/bin/google-chrome
 TargetEnvironment=Unity
 
 [NewIncognito Shortcut Group]
@@ -220,3 +220,3 @@
 Name[zh_TW]=新增無痕式視窗
-Exec=/usr/bin/google-chrome-stable --incognito
+Exec=/usr/bin/google-chrome --incognito
 TargetEnvironment=Unity
